### ISSUE TITLE
Is this Library still mentained?

### ISSUE THREAD

**Issue by GenGraphic** [MrHertal/react-native-twilio-phone#129]
I m having some issues with receiveing the Calls. I m dealing with this for about a week, and i would like to know if there is still someone that can give me a hand,. Thank  you!

**Comment by MrHertal** [comment]
No, use this instead: https://github.com/twilio/twilio-voice-react-native
